SmartDeviceLink/public/SDLChoiceSet.m

```diff
@@ -57,7 +58,9 @@ - (instancetype)initWithTitle:(NSString *)title delegate:(id<SDLChoiceSetDelegat
         return nil;
     }
 
-    if (timeout < 5 || timeout > 100) {
+    if (timeout == 0) {
+        SDLLogV(@"Creating a choice set with a 0 second timeout; the default timeout will be used: %f seconds", SDLChoiceSet.defaultTimeout);
```

Comment:
```suggestion
        SDLLogV(@"Creating a choice set with a 0 second timeout; the default timeout will be used: %.01f seconds", SDLChoiceSet.defaultTimeout);
```

This probably needs to reduce the number of decimals.

Reply:
This log was removed